.eslintrc.json

```diff
@@ -6,6 +6,7 @@
     "plugin:github/typescript"
   ],
   "globals": {
-    "CustomElementElement": "readonly"
+    "CustomElementElement": "readonly",
+    "module": true
```

Comment:
PS: also added this to prevent `module.exports` throwing a lint error

Reply:
What lint error are you getting. When I remove this locally and run `npm test` I don't get any errors.

Reply:
This is the error I get:
<img width="503" alt="Screen Shot 2021-05-28 at 11 20 11 AM" src="https://user-images.githubusercontent.com/5520141/120013964-15232300-bfa7-11eb-8e8f-79b9bfb5e402.png">